src/Client/App.fs

```diff
@@ -143,36 +135,15 @@ open Client.Style
 
 /// Constructs the view for a page given the model and dispatcher.
 let viewPage model dispatch =
-    match model.Page with
-    | Page.Home ->
-        [ viewLink Page.Login "Please login into the SAFE-Stack sample app"
-          br []
-          br []
-          br []
-          br []
-          br []
-          br []
-          br []
-          words 20 "Made with"
-          a [ Href "https://safe-stack.github.io/" ] [ img [ Src "/Images/safe_logo.png" ] ]
-          words 15 "An end-to-end, functional-first stack for cloud-ready web development that emphasises type-safe programming."
-          br []
-          br []
-          br []
-          br []
-          words 20 ("version " + ReleaseNotes.Version) ]
-
-    | Page.Login ->
-        match model.SubModel with
-        | LoginModel m ->
-            [ div [ ] [ Login.view m (LoginMsg >> dispatch) ]]
-        | _ -> [ ]
-
-    | Page.WishList ->
-        match model.SubModel with
-        | WishListModel m ->
-            [ div [ ] [ WishList.view m (WishListMsg >> dispatch) ]]
-        | _ -> [ ]
+    match model.PageModel with
+    | HomePageModel ->
+        Home.view ()
```

Comment:
`site.css` edits needed in order to surround this in a `div` like the other cases,

Reply:
Maybe the div can be moved out